app/models/group.rb

```diff
@@ -3,6 +3,8 @@
 require "net/imap"
 
 class Group < ActiveRecord::Base
+  MAX_EMAIL_DOMAIN_LENGTH = 253
```

Comment:
This is the longest a canonicalized domain name can be.

Reply:
A comment with a link to that section of the email RFC would be useful here :)